Sources/App/Commands/ReconcilePackageListCommand.swift

```diff
@@ -29,18 +29,20 @@ final class ReconcilePackageListCommand: Command
         
         // Delete any removed packages
         let packageDeletions = reconciler.packagesToDelete.map { url in
-          return Package.findByUrl(on: database, url: url).map { package -> Future<Void> in
+          return Package.findByUrl(on: database, url: url).flatMap { package -> Future<Void> in
             print("Deleting \(package.url)")
             return package.delete(on: database).map {
               print("Deleted \(package.url)")
             }
           }
         }
-
+        
         // Queue up all of the additions and deletions for execution
-        return packageAdditions.flatten(on: database)
-          .and(packageDeletions.flatten(on: database))
-          .transform(to: ())
+        
+        return map(packageAdditions.flatten(on: database), packageDeletions.flatten(on: database)) { additions, _ in
+          print("Added \(additions.count) additions")
+          print("Finished")
+        }
```

Comment:
Is there an advantage to doing it this way over using `add`?

The cleanliness I can see is that you no longer need the `transform`, is it as simple as that?

Reply:
No more habit. You could do it with `transform` as well, but I removed that to add the print statements.

The advantage of using the variadic `map` or `flatMap` over `and` is that you don't end up in tuple hell when chaining more than 2 futures together

Reply:
> you don't end up in tuple hell when chaining more than 2 futures together 

I totally see that, thanks.